sdk/servicebus/Azure.Messaging.ServiceBus/src/Addons/ServiceBusClientExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Azure.Messaging.ServiceBus.Addons
```

Comment:
I'd prefer that we keep each transport protocol isolated in its own namespace.   I can only use one protocol at a time, so I shouldn't see `AMQP` and `MQTT` extensions at the same time, for example.

Reply:
I moved these extensions methods to `Azure.Messaging.ServiceBus.Transports.Amqp`.